README.md

```diff
@@ -37,6 +37,7 @@ You can configure the default behavior through various settings in your `setting
   "newFile.rootDirectory": "~",
   "newFile.showPathRelativeTo": "root", // "project" or "none"
   "newFile.expandBraces": false // setting to true will allow for creating multiple files such as `new-folder/{file1,file2}.js`
+  "workbench.quickOpen.closeOnFocusLost": false // setting to true, quick open should close automatically once it loses focus
```

Comment:
Shouldn't we add something like a "Frequently asked" section where this would belong?

Reply:
cc @dkundel

Reply:
I actually thought of the same but went with this since its related to this. Do you want me to change it ?

Reply:
Yes because it's not directly related to the extension. It's more like a nice to have which will impact several extensions at once.

Reply:
Yeah I agree it should go rather into a frequently asked questions section. It's modifies the behavior of more than just this extension and therefore shouldn't create the idea for users that this is part of the extension.

Reply:
Done with it. Can you take a look at the above change in the versioning too because I think it changed when I did `npm install`❓